docs/content/docs/operations/upgrade.md

```diff
@@ -59,6 +59,16 @@ Please note that we are using the `replace` command here which ensures that runn
 
 ### 2. Upgrading the Helm deployment
 
+{{< hint danger >}}
+Before upgrading, please compare the version difference between the currently generated yaml and the running yaml, which will be used for backup and restore.
+{{< /hint >}}
+
+
+```sh
+helm template flink-kubernetes-operator <helm-repo>/flink-kubernetes-operator  <custom settings> | kubectl diff -f -
+```
```

Comment:
What is the point of this step?

Reply:
We hope to compare the version difference between the current version and the production environment to avoid misuse.

Reply:
Would like an extra step to check, for example if the user has already upgraded, or if the upgrade was done from v1alpha1 version.

Reply:
Thank you for taking the time to review and respond!